pkg/controller/job/job_controller.go

```diff
@@ -1384,6 +1393,7 @@ func jobSuspended(job *batch.Job) bool {
 func (jm *Controller) manageJob(ctx context.Context, job *batch.Job, jobCtx *syncJobCtx) (int32, string, error) {
 	logger := klog.FromContext(ctx)
 	active := int32(len(jobCtx.activePods))
+	terminating := int32(len(jobCtx.terminatingPods))
```

Comment:
It seems used in one place only, so I prefer to inline it, especially as here it is not wrapped in gates check.

Reply:
I gated the addition of terminatingPods so if cases where this is disabled this would be 0 so it would be a no-opt.

Reply:
Hm two things: 
- I think gating every use of new vars is good is possible, so that it is clear to the reader that this alpha feature does not break something and it can be disabled
- I think this list if only needed for indexed jobs, so no need to waste memory on it, unless I'm missing something, IIUC it is enough to compute it only in the firstPendingIndexes function.

Reply:
good point.  added.